doc/source/reference/random/performance.rst

```diff
@@ -42,49 +42,49 @@ Integer performance has a similar ordering.
 
 The pattern is similar for other, more complex generators. The normal
 performance of the legacy `RandomState` generator is much
-lower than the other since it uses the Box-Muller transformation rather
-than the Ziggurat generator. The performance gap for Exponentials is also
+lower than the other since it uses the Box-Muller transform rather
+than the Ziggurat method. The performance gap for Exponentials is also
 large due to the cost of computing the log function to invert the CDF.
-The column labeled MT19973 is used the same 32-bit generator as
-`RandomState` but produces random values using
-`Generator`.
+The column labeled MT19973 uses the same 32-bit generator as
+`RandomState` but produces random variates using `Generator`.
 
 .. csv-table::
-    :header: ,MT19937,PCG64,Philox,SFC64,RandomState
+    :header: ,MT19937,PCG64,PCG64DXSM,Philox,SFC64
     :widths: 14,14,14,14,14,14
 
-    32-bit Unsigned Ints,3.2,2.7,4.9,2.7,3.2
-    64-bit Unsigned Ints,5.6,3.7,6.3,2.9,5.7
-    Uniforms,7.3,4.1,8.1,3.1,7.3
-    Normals,13.1,10.2,13.5,7.8,34.6
-    Exponentials,7.9,5.4,8.5,4.1,40.3
-    Gammas,34.8,28.0,34.7,25.1,58.1
-    Binomials,25.0,21.4,26.1,19.5,25.2
-    Laplaces,45.1,40.7,45.5,38.1,45.6
-    Poissons,67.6,52.4,69.2,46.4,78.1
+    32-bit Unsigned Ints,94,168,152,93,176
+    64-bit Unsigned Ints,98,174,180,109,209
+    Uniforms,97,192,197,112,224
+    Normals,333,418,406,341,535
+    Exponentials,676,956,937,622,973
+    Gammas,297,287,280,216,292
+    Binomials,100,123,122,107,138
+    Laplaces,149,200,183,113,105
+    Poissons,47,59,61,47,52
```

Comment:
Hmm, how did our Poissons get so slow?

Reply:
I don't see that, for instance.

Reply:
I suspect this is AMD-specific.  Both Laplaces and Poissions are slower, although the relative performance to RandomState hasn't changed much (it is also slower).